<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysInfoFrontClass</class>
 <widget class="QMainWindow" name="SysInfoFrontClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SysInfoFront</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="view_table">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>1051</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="proc_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Processes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disk_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Disks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="netw_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Network</string>
    </property>
   </widget>
   <widget class="QPushButton" name="device_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Devices</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1179</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sysinfofront.qrc"/>
 </resources>
 <connections/>
</ui>
